assets/src/ba_data/python/bastd/game/capturetheflag.py

```diff
@@ -63,7 +63,8 @@ def reset_return_times(self) -> None:
         self.touch_return_time = float(
             self.activity.settings['Flag Touch Return Time'])
 
-    def get_team(self) -> ba.Team:
+    @property
+    def team(self) -> ba.Team:
```

Comment:
Nice! I am trying to migrate things like this to properties since those are more elegant looking in code..